english/statue/dont-blink/dont-blink.md

```diff
@@ -0,0 +1,17 @@
+Your eyes burn and itch. You can't stand it anymore.
+    Not. One. More. Second.
+        You blink.
+
+The statue crumbles, revealing a thin man in a dark suit
+and trainers. "Brilliant!" He says, grinning like a mad man.
+"Lucky for you," he says, brushing dust from his sleeve and
+hopping around like a bird. "I'm no angel."
+
+He runs to the fridge and throws open the door.
+"Come on," he says. "Let's go!"
+
+[Believe the lie.](../the-smoking-man/believe/believe.md)
+
+[You say, "Uh, no. It's dangerous in there, take this," and hand him a spork from the top of an oddly disturbing, overflowing trash can.](../the-cake/dangerous/dangerous.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.